demos/motion_tracking/motion_tracking_network.py

```diff
@@ -0,0 +1,229 @@
+# Copyright (C) 2023 Intel Corporation
```

Comment:
Suggesting to rename module to `motion_tracker`.

Reply:
Done